packages/cli/src/commands/create.ts

```diff
@@ -26,3 +25,22 @@ export const createNewContentBlock = (contentBlockName: string, template = 'tail
 
     Logger.defaultInfo(`\n${Logger.spacer(11)}Happy hacking!`);
 };
+export const createNewApp = (appName: string, template, type: string): void => {
```

Comment:
```suggestion
};

export const createNewPlatformApp = (appName: string, template, type: string): void => {
```

Reply:
Ive just seen, these 2 things are so much the same, they should not coexist.. if its fine that a response in the cli says that the "Scaffolding App..." instead of "Scaffolding Block" then we could just use only one method